app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_comments_counter(count)
+    post.update(comment_counter: count)
+  end
 end
```

Comment:
- [ ] Kindly consider making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to the database, then make the `update_comments_counter` method private

e.g

```ruby
 after_save :update_comments_counter

  private

  def update_comments_counter
    author.increment!(:comment_counter)
  end
```